recipe/meta.yaml

```diff
@@ -6,48 +6,84 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/tfcausalimpact-{{ version }}.tar.gz
-  sha256: 0e362965082377a3518f56b555ede56e4fedefd3f44fd03cd40946470ff75375
+  url: https://github.com/WillianFuks/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3e2c588b2bac36557c9b2aefd30c15e0e2ccfa4609828b47c70612a4b5632490
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: True  # [py>312]
 
 requirements:
   host:
-    - python {{ python_min }}
-    - flake8
-    - isort
     - pip
+    - python
     - setuptools
+    - wheel
+    - isort
+    - flake8
   run:
-    - python >={{ python_min }},<3.13
+    - python
     - jinja2
     - pandas >=1.3.5,<=2.2
     - tensorflow >=2.10
     - tensorflow-probability >=0.18,<=0.25
+    # tensorflow-probability[tf]
+    - tf-keras >=2.16
     - matplotlib-base
 
-    # tensorflow-probaility[tf]
-    - tf-keras >=2.10
-
 test:
+  files:
+    - tests
+  source_files:
+    - tests
   imports:
     - causalimpact
+    - causalimpact.__version__
+    - causalimpact.data
+    - causalimpact.inferences
+    - causalimpact.main
+    - causalimpact.misc
+    - causalimpact.model
+    - causalimpact.plot
+    - causalimpact.summary
   commands:
-    - pip check
+    # Skip pip check on Windows due to conda/pip metadata conflict:
+    # tf-keras conda package references pip tensorflow dependency but we have conda tensorflow
+    - pip check  # [not win]
```

Comment:
Can you check the output of `python -c "import keras._tf_keras.keras.backend as K; print(K)"`?

Reply:
Okay, it prints `Keras backend= torch`. At least, the issue is visible now.